.github/workflows/publish.yml

```diff
@@ -0,0 +1,20 @@
+name: Publish Docker Image
+
+on:
+  schedule:
+    - cron:  '0 0 * * *'
```

Comment:
Should we run it during daylight hours EST or PST so that people can fix it quicker should it break?

Reply:
Good question. I think that's smart.